my-app/src/main/java/com/mycompany/app/App.java

```diff
@@ -25,5 +26,6 @@ public static void main( String[] args )
          * http://checkerframework.org
          */
         System.out.println("... but thankfully, Checker has our back: http://checkerframework.org");
+         System.out.println("... but thankfully, Checker has our back: http://checkerframework.org");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ec2-54-186-68-134.us-west-2.compute.amazonaws.com:9000/coding_rules#rule_key=squid%3AS106)